packages/lit-dev-content/site/tutorials/content/svg-templates/04.md

```diff
@@ -0,0 +1,84 @@
+### Overview
+
+This is chapter of the demo covers the basics of patterns in SVG
+including how to:
+
+- create a `<pattern>` 
+- apply a `<pattern>` as a fill
+
+### Learn
+
+In SVG, the `<pattern>` element creates a tile that can be repeated. 
+
+The `patternUnits` property defines the relative coordinate geometry
+of the pattern.
+
+Setting `patternunits` as `userSpaceOnUse` makes geometry relative
+to the user space. In this case, user space is the DOM.
+
+```TS
+const helloPattern = svg`
+	<pattern patternUnits="userSpaceOnUse">
+		${createTile()}
+	</pattern>
+`;
+```
+
+The `<pattern>` element can be given an ID and referenced as a fill
+in other svg elements.
+
+```TS
+const helloPattern = svg`
+	<pattern id="pattern-rounds" patternUnits="userSpaceOnUse">
+		${createTile()}
+	</pattern>
+`;
+
+const helloPatternFill - svg`
+    <rect fill="url(#pattern-rounds)" width="200" height="200"></rect>
+`;
+```
+
+# Apply
```

Comment:
This heading is inconsistent with previous steps

Reply:
done!